tests/integration/golang/aim/app/delete_app_test.go

```diff
@@ -0,0 +1,114 @@
+//go:build integration
+package run
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type DeleteAppTestSuite struct {
+	suite.Suite
+	client      *helpers.HttpClient
+	appFixtures *fixtures.AppFixtures
+	apps        []*models.App
+}
+
+func TestDeleteAppTestSuite(t *testing.T) {
+	suite.Run(t, new(DeleteAppTestSuite))
+}
+
+func (s *DeleteAppTestSuite) SetupTest() {
+	s.client = helpers.NewAimApiClient(os.Getenv("SERVICE_BASE_URL"))
+
+	appFixtures, err := fixtures.NewAppFixtures(os.Getenv("DATABASE_DSN"))
+	assert.Nil(s.T(), err)
+	s.appFixtures = appFixtures
+
+	s.apps, err = s.appFixtures.CreateTestApps(context.Background(), 10)
```

Comment:
do we really need to create 10 apps? isn't enough to create just one app, then delete it and check that there is nothing in database right after that?

Reply:
it is a but confused to create 10 apps but we only need only 1.

Reply:
okay will reduce to 1